README.md

```diff
@@ -163,6 +163,82 @@ back up:
 kubectl -n YOUR_POD_NAMESPACE annotate pod/YOUR_POD_NAME backup.velero.io/backup-volumes=FILE_VOLUME_NAME_1,FILE_VOLUME_NAME_2,...
 ```
 
+## vSphere with Kubernetes notes
+### Restricted resources
+Certain resources with the vSphere with Kubernetes Supervisor Cluster need to be created by the Supervisor Cluster and cannot be restored.  The Velero Plugin for vSphere blocks
+backup and restore of these resources and will generate errors if an attempt is made to backup or restore these resources and the backup or restore will be marked as
+"Partially Failed".  To avoid these errors, exclude the resources from your backups.  The current list of blocked resources is:
```

Comment:
Sure.  I added them to our own resources because we own those.  Having the exclude tag means that we won't get errors if someone tries to backup the Velero space.